test/jdk/javax/swing/JFileChooser/FileChooserSymLinkTest.java

```diff
@@ -70,13 +71,13 @@ static void initialize() throws InterruptedException, InvocationTargetException
         final String INSTRUCTIONS = """
                 Instructions to Test:
                 1. Create a regular directory in any specific path.
-                    ex: mkdir c:\\target
+                    ex: mkdir C:\\target
                 2. Create a Symbolic link targeting the created test directory.
-                    ex : mklink /D c:\\link c:\\target
+                    ex : mklink /D C:\\link C:\\target
                 3. In JFileChooser, navigate to "link" created directed.
                 4. On click of the "link" directory, if the Absolute path of
                     Symbolic Link is valid then Click PASS, else if it is
-                    null then Click FAIL.
+                    null then Click FAIL.(Including MultiSelection Mode SET)
```

Comment:
Is this possible? Your test does not allow for multi-selection mode.

Reply:
Should I enable multiselection in code?